src/map/unit.cpp

```diff
@@ -481,6 +481,14 @@ static TIMER_FUNC(unit_walktoxy_timer)
 		}
 #endif
 
+		// Remove any possible escape states present for mobs once they stopped moving.
+		if (bl->type == BL_MOB) {
+			mob_data *md = BL_CAST(BL_MOB, bl);
```

Comment:
No need to BL_CAST anymore, since you already checked the type in the line above.

Reply:
If I read up a little more in the function I would have seen `md` is already defined too. 🙃